addon/model.js

```diff
@@ -119,7 +119,6 @@ function resolveValue(key, value, modelName, store, schema, model, valueInArray
       // internally within ember-data
       modelName: nested.type ? dasherize(nested.type) : null,
       attributes: nested.attributes,
-      store,
```

Comment:
good to remove this, but we should remove it from `EmbeddedInternalModel`'s constructor as well

Reply:
Removed there as well.